File: javascript-closure-angular-client/API/Client/SalesHeaderInPayment.js
goog.provide('API.Client.SalesHeaderIn_payment');

/**
 * @record
 */
API.Client.SalesHeaderInPayment = function() {}

/**
 * @type {!API.Client.PaymentTerms}
 * @export
 */
API.Client.SalesHeaderInPayment.prototype.terms;

/**
 * installments
 * @type {!Array<!API.Client.SalesInstallmentIn>}
 * @export
 */
API.Client.SalesHeaderInPayment.prototype.installments;

File: dot-net-client/src/Avalara.AvaTax.Brazil.RestClient/Model/SalesHeaderInPayment.cs
using Avalara.AvaTax.Brazil.RestClient.Model.Enum;
using System.Collections.Generic;
using System.Runtime.Serialization;

namespace Avalara.AvaTax.Brazil.RestClient.Model
{
    /// <summary>
    /// SalesHeaderInPayment
    /// </summary>
    [DataContract]
    public class SalesHeaderInPayment
    {
        /// <summary>
        /// Gets or Sets Terms
        /// </summary>
        [DataMember(Name = "terms", EmitDefaultValue = false)]
        public PaymentTerms Terms { get; set; }

        /// <summary>
        /// installments
        /// </summary>
        /// <value>installments</value>
        [DataMember(Name = "installments", EmitDefaultValue = false)]
        public List<SalesInstallmentIn> Installments { get; set; }
    }
}